### DISCUSSION TITLE
Why was the Kryptorremoved from privacytools

### DISCUSSION THREAD

**Discussion by sidielnikov** [samuel-lucas6/Kryptor#48] [General]
Hello why the Kryptorwas removed from privacytools.ioin the Security File Encryption? (www.privacytools.io/secure-file-encryption/)

**Comment by samuel-lucas6** [comment]
I have no idea. The team behind PrivacyTools became [PrivacyGuides](https://www.privacyguides.org/) because the founder wasn't playing ball. Now the founder has become active again and has added nonsense recommendations (e.g. NordVPN, NordPass, etc because of affiliate links).

Thus, follow PrivacyGuides instead and treat PrivacyTools as someone just listing off a bunch of software without evaluating it properly.
